### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by quantshah**
I updated the readme to give some more information. We need to decide and add a LICENSE (probably compatible with QuTiP's LICENSE).

### CODE REVIEW

**Comment by quantshah** [comment]
Let's practice a simple review. It is good policy to not merge your own code changes on a project and always open Pull requests so that others can give comments. 

The best practice is always to open a new PR using a fresh branch to avoid conflicts and "NEVER" working on the master. But in this case, since it is just the README, I will let myself off the hook.

Feel free to give any comments and then approve merging the changes.

**Comment by nathanshammah** [comment]
Thanks @quantshah. Let's use `Reviewers` for review of someone else's PR, while `Assignees` are in charge of getting the task done (also with Issues). When one is pinged for a review, he/she can use the review tool, and add a series of comments to the PR, which then need to be addressed by the PR owner; these comments can include [suggesting commits](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) from the web interface.

**Comment by nathanshammah** [comment]
@quantshah or @araza6 please update my access to the repository, so that I can also add / self assign PRs and issues, as well as adding labels.

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# qgrad
\ No newline at end of file
+# qgrad
+
+A Python library to integrate automatic differentiation tools such as Jax with QuTiP and related quantum software packages.
+
+This package is a work in progress. Feel free to take part in the discussions by opening new issues.
+
+# Contributing
```
Comment (nathanshammah) [inline README.md:7]:
```suggestion
## Contributing
```

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+We are in the early stages of desining the tool and welcome any discussion in the form of issues or Pull requests.
```
Comment (nathanshammah) [inline README.md:8]:
```suggestion
We are in the early stages of designing the tool and welcome any discussion in the form of [Issues](https://github.com/qgrad/qgrad/issues/new) or Pull Requests. 
```

**Comment by araza6** [comment]
> @quantshah or @araza6 please update my access to the repository, so that I can also add / self assign PRs and issues, as well as adding labels.

Hi Nathan, you are the owner now

**Comment by quantshah** [comment]
Thanks for the comments @nathanshammah. How do we convert a PR to draft? I would say this PR is merge-ready now.

**Comment by nathanshammah** [comment]
@quantshah you have the option when opening it. Now you can also convert it, see under Reviewers on the top right. I am also fine with keeping them with "WIP: " in the title if it is simpler, and change the title then; we could do both ways.

**Pull request merged**